client/src/components/my-mentor-card/recommended-action-button.tsx

```diff
@@ -5,20 +5,29 @@ Permission to use, copy, modify, and distribute this software and its documentat
 The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
 */
 import { Button, Typography } from "@material-ui/core";
-import { DoubleArrow, PlayCircleFilled } from "@material-ui/icons";
+import { DoubleArrow } from "@material-ui/icons";
 import React from "react";
 import { Mentor } from "types";
 import { UseWithRecommendedAction } from "./use-with-recommended-action";
 export default function RecommendedActionButton(props: {
   setThumbnail: (file: File) => void;
+  buildAction: (mentorId: string) => void;
```

Comment:
same comment...I suspect `buildAction` should not require or accept a mentor param (because mentor is known and fixed)